kmip/services/server/crypto/engine.py

```diff
@@ -868,3 +884,123 @@ def wrap_key(self,
                 "Wrapping method '{0}' is not a supported key wrapping "
                 "method.".format(wrapping_method)
             )
+
+    def _create_RSA_private_key(self,
+                                bytes):
+        """
+        Instantiates an RSA key from bytes.
+
+        Args:
+            bytes (byte string): Bytes of public or private RSA key.
+            private (boolean): Indicates whether public or private key.
```

Comment:
The `private` argument can be removed here, unless you plan on adding that capability to the method.